addons/account/models/account_tax.py

```diff
@@ -381,6 +382,9 @@ def _compute_display_name(self):
                 name += ' (%s)' % type_tax_use.get(record.type_tax_use)
             if record.tax_scope:
                 name += ' (%s)' % tax_scope.get(record.tax_scope)
+            if record.company_id.multi_vat_foreign_country_ids\
+                    and record.country_id != record.company_id.account_fiscal_country_id:
```

Comment:
Not sure why you want to check `multi_vat_foreign_country_ids`
```suggestion
            if record.country_id != record.company_id.account_fiscal_country_id:
```

Reply:
I was about to say "to not have that appended on OSS taxes which would be a doublon" but checking it, it seems that all OSS taxes are created with the `account_fiscal_country_id`.
You learn new stuff everyday. :D
I'll remove it!

Thanks for the catch!